### PR TITLE
Pd 605 606 607 helm chart and ui

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/use-px-backup/uninstall.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Uninstall PX-Backup
+description: Upgrade your version of PX-Backup
```
Comment (bhavana-ar) [inline content/use-px-backup/uninstall.md]: Description should be uninstall PX-backup and not Upgrade

File: content/use-px-backup/configure-ui.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+title: Configure access to the PX-Backup UI
+description: Configure access to the PX-Backup UI
+keywords: upgrade,
```
Comment (bhavana-ar) [inline content/use-px-backup/configure-ui.md:4]: Keyword says upgrade. But this section is for configuring and accessing UI

File: content/use-px-backup/configure-ui.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+description: Configure access to the PX-Backup UI
+keywords: configure,
+weight: 10
+hidesections: true
+disableprevnext: true
+scrollspy-container: false
+type: common-landing
+---
+
+By default, the PX-Backup UI is configured <!-- need description of the current default config here. -->. If the standard PX-Backup UI configuration doesn't work with your cluster architecture, you can alter it so that it does.
```
Comment (nathan-px) [inline content/use-px-backup/configure-ui.md:12]: @bhavana-ar @tusharraut1994 I think we need more concept information here. What does the default config result in? I'm thinking of changing it to something like:

```suggestion
By default, you can access the PX-Backup UI through an HTTP endpoint  <!-- need to describe the standard behevior here -->. If the standard PX-Backup UI endpoint configuration doesn't meet your requirements, you can configure access using HTTPS, access it through the load balancer, or even navigate to one of your node IPs directly. ```

File: content/use-px-backup/configure-ui.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+By default, the PX-Backup UI is configured <!-- need description of the current default config here. -->. If the standard PX-Backup UI configuration doesn't work with your cluster architecture, you can alter it so that it does. 
+
+<!-- Revisit this whole description -->
+
+## Expose the PX-Backup UI on ingress and configure access using HTTPS
+
+You can configure access to PX-Backup through HTTPS by creating an ingress rule.
+
+1. Create the following spec, entering your own secret and hosts based on your cluster's configuration:<!-- What do they need to enter here? I assume `test`, but what else? maybe this value? px-backup-ui.test-1.us-east.containers.appdomain.cloud . I think this is also an unnecessarily compounded command piping the spec through cat creates the spec upon pasting -- which makes step 2 irrelevant -->
```
Comment (nathan-px) [inline content/use-px-backup/configure-ui.md:20]: @bhavana-ar @tusharraut1994 What do they need to enter here? I assume `test`, but what else? maybe this value? px-backup-ui.test-1.us-east.containers.appdomain.cloud . I think this is also an unnecessarily compounded command piping the spec through cat creates the spec upon pasting -- which makes step 2 irrelevant

**Pull request merged**